sceptre/exceptions.py

```diff
@@ -7,6 +7,12 @@ class SceptreException(Exception):
     """
 
 
+class ProjectExistsError(SceptreException):
+    """
+    Error raised when Sceptre project already exists.
+    """
+
```

Comment:
It looks like this is the only error which doesn't call pass, is there a reason for this?

Reply:
You're right it should!